Make Dataset/scripts/bin_to_hfive.py

```diff
@@ -0,0 +1,69 @@
+# Enter $path_to_set of depth maps 
+# and $output_file.
+
+import h5py
+import numpy as np
+from scipy import ndimage
+ 
+import sys
+import os
+import struct
+from pathlib import Path
+
+
+def read_array(path: os.PathLike, window_size: int = 5) -> np.ndarray:
+    """Converting from .bin to .h5 function.
+
+    Extract ndarray from entered .bin file,
+    preproccess it with smoothing aloritm 
+    and rewrite in .h5 format. 
+    """
+
+    with open(path, "rb") as fid:
+        width, height, channels = np.genfromtxt(fid, delimiter="&", max_rows=1,
+                                                usecols=(0, 1, 2), dtype=int)
+        fid.seek(0)
+        num_delimiter = 0
+        byte = fid.read(1)
+
+        while True:
+            if byte == b"&":
+                num_delimiter += 1
+                if num_delimiter >= 3:
+                    break
+            byte = fid.read(1)
+
+        array = np.fromfile(fid, np.float32)
+
+    array = array.reshape((width, height, channels), order="F")
+    array = np.transpose(array, (1, 0, 2)).squeeze()
+
+    return ndimage.median_filter(array, size=window_size)
+
+
+def main():
+
+    inp_dir = Path(sys.argv[1]) if len(sys.argv) > 1 else Path("./")
```

Comment:
Use [argparse](https://docs.python.org/3/library/argparse.html#module-argparse) to make script friendly